app/controllers/admin/creator_settings_controller.rb

```diff
@@ -12,13 +13,10 @@ def create
         ::Settings::UserExperience.primary_brand_color_hex = settings_params[:primary_brand_color_hex]
         ::Settings::Authentication.invite_only_mode = settings_params[:invite_only]
         ::Settings::UserExperience.public = settings_params[:public]
+        settings_params[:checked_code_of_conduct]
+        settings_params[:checked_terms_and_conditions]
       end
-      # For this feature to work as expected for the time being, we must set the COC and TOS to true.
-      # However, this is not a viable solution, as Forem Creators are required to see and check the
-      # COC and TOS. Bypassing them in this manner will not do and we will need to rethink this solution.
-      # TODO: Replace the current solution of setting the COC and TOS to true with a better, more
-      # long-term solution for Forem Creators.
-      current_user.update!(saw_onboarding: true, checked_code_of_conduct: true, checked_terms_and_conditions: true)
+      current_user.update!(saw_onboarding: true)
```

Comment:
We can the add them here where once we've saved whatever we need to to settings, we can then go about updating the current user :) 

```suggestion
      current_user.update!(
        saw_onboarding: true,
        checked_code_of_conduct: settings_params[:checked_code_of_conduct],
        checked_terms_and_conditions: settings_params[:checked_terms_and_conditions]
      )
```

Reply:
OMG, duh, me! 🤦‍♀️  Thank you, thank you, thank you, @Ridhwana!

Reply:
It's a pleasure! I didn't notice it initially either 😅